Tasks/AzureCLIV2/src/ScriptType.ts

```diff
@@ -7,14 +7,14 @@ export class ScriptTypeFactory {
         let scriptLocation: string = tl.getInput("scriptLocation", true);
```

Comment:
in yaml case, should we check scriptLocation is non-empty and value is ["scriptpath", "inlineScript"] else stop execution

Reply:
done